less/opendev.less

```diff
@@ -63,6 +63,16 @@ td{
   box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.8);
 }
 
+#mekong-color, .mekong-color{
```

Comment:
@Huyeng this is defined on https://github.com/OpenDevelopmentMekong/wp-odm_theme/blob/master/less/_colors.less#L53. I think it suits better there